Resources/templates/aliases.php.twig

```diff
@@ -1,13 +1,20 @@
 <?php
 
+{% for alias in aliases|slice(0,1) %}
+/**
+ * @file
+ * Drush site-alias configuration for {{ aliases["ac-site"] }}.
```

Comment:
Perhaps for now I could just simplify this comment and leave out the `ac-site` or `site` keys. Identifying it in the `@file` doc comment is a _nice to have_, but not strictly necessary.